src/api/Tag.php

```diff
@@ -15,7 +15,8 @@ class Tag
     const RESOURCE_NAME = 'resource.name';
     const DB_STATEMENT = 'sql.query';
     const ERROR = 'error';
-    const ERROR_MSG = 'error.msg'; // string representing the error message
+    const ERROR_MESSAGE = 'error.message'; // string representing the error message
+    const ERROR_MSG = 'error.msg';/** @deprecated Use ERROR_MESSAGE instead. */
```

Comment:
Documentation comments need to be placed _before_ the thing they apply to. I know it's not consistent with the comment style being used right here, but that's how it goes.

As written, this `@deprecated` would apply to `ERROR_TYPE`.

Reply:
Good to know! Just fixed this